docker/orientdb/init/create-db-with-schema.osql

```diff
@@ -0,0 +1,232 @@
+CREATE DATABASE remote:localhost/kilda root root PLOCAL GRAPH;
+
+CREATE CLASS kilda_configuration IF NOT EXISTS EXTENDS V;
+CREATE PROPERTY kilda_configuration.unique IF NOT EXISTS STRING;
+CREATE INDEX kilda_configuration.unique UNIQUE_HASH_INDEX;
+
+CREATE CLASS feature_toggles IF NOT EXISTS EXTENDS V;
+CREATE PROPERTY feature_toggles.unique IF NOT EXISTS STRING;
+CREATE INDEX feature_toggles.unique UNIQUE_HASH_INDEX;
+
+CREATE CLASS switch IF NOT EXISTS EXTENDS V;
+CREATE PROPERTY switch.name IF NOT EXISTS STRING;
+CREATE PROPERTY switch.state IF NOT EXISTS STRING;
+CREATE PROPERTY switch.features IF NOT EXISTS EMBEDDEDSET STRING;
+CREATE INDEX switch.name UNIQUE_HASH_INDEX;
+CREATE INDEX switch.state NOTUNIQUE_HASH_INDEX;
+
+CREATE CLASS switch_connected_device IF NOT EXISTS EXTENDS V;
+CREATE PROPERTY switch_connected_device.flow_id IF NOT EXISTS STRING;
+CREATE PROPERTY switch_connected_device.switch_id IF NOT EXISTS STRING;
+CREATE PROPERTY switch_connected_device.port_number IF NOT EXISTS INTEGER;
+CREATE PROPERTY switch_connected_device.vlan IF NOT EXISTS INTEGER;
+CREATE PROPERTY switch_connected_device.mac_address IF NOT EXISTS STRING;
+CREATE PROPERTY switch_connected_device.type IF NOT EXISTS STRING;
+CREATE PROPERTY switch_connected_device.chassis_id IF NOT EXISTS STRING;
+CREATE PROPERTY switch_connected_device.port_id IF NOT EXISTS STRING;
+CREATE PROPERTY switch_connected_device.ip_address IF NOT EXISTS STRING;
+CREATE INDEX switch_connected_device.flow_id NOTUNIQUE_HASH_INDEX;
+CREATE INDEX switch_connected_device.switch_id NOTUNIQUE_HASH_INDEX;
+CREATE INDEX switch_connected_device.port_number NOTUNIQUE_HASH_INDEX;
+CREATE INDEX switch_connected_device.vlan NOTUNIQUE_HASH_INDEX;
+CREATE INDEX switch_connected_device.mac_address NOTUNIQUE_HASH_INDEX;
+CREATE INDEX switch_connected_device.type NOTUNIQUE_HASH_INDEX;
+CREATE INDEX switch_connected_device.chassis_id NOTUNIQUE_HASH_INDEX;
+CREATE INDEX switch_connected_device.port_id NOTUNIQUE_HASH_INDEX;
+CREATE INDEX switch_connected_device.ip_address NOTUNIQUE_HASH_INDEX;
```

Comment:
Why we do not have a unique index for field group? like for flow_meter:
```CREATE INDEX flow_meter_unique on flow_meter (switch_id, meter_id) UNIQUE;```

Previously it was something like 
```  
private void calculateUniqueIndex() {
        switch (type) {
            case LLDP:
                uniqueIndex = format("%s_%s_%s_%s_%s_%s_%s",
                        switchObj.getSwitchId(), portNumber, vlan, macAddress, type, chassisId, portId);
                break;
            case ARP:
                uniqueIndex = format("%s_%s_%s_%s_%s_%s",
                        switchObj.getSwitchId(), portNumber, vlan, macAddress, type, ipAddress);
                break;
            default:
                throw new IllegalArgumentException(format("Unknown connected device type %s", type));
        }
    }
```

Reply:
Agree. The index should be re-introduced. The issue has been reported - #3760